jest.setup.ts

```
@@ -1 +1,4 @@
 import "@testing-library/jest-dom"
+
+require('dotenv').config();
+setupFiles: ['<rootDir>/setup-jest.js']
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)